UI/js/modals.js

```diff
@@ -29,4 +29,8 @@ const deleteAnnouncement = () => {
       modal.style.display = "none";
     }
   }
-}
\ No newline at end of file
+}
+// User flag announcement
+const flagAnnouncement = () => {
+  alert("Announcement flagged successfully!");
+}
```

Comment:
The same function is in `UI/js/feed.js`, what do you think by removing one?? and remain with the one that is used

Reply:
Yes. It makes sense since they are both doing the same thing (notifying the user). Thanks.